packages/angular_devkit/build_angular/src/builders/application/execute-post-bundle.ts

```diff
@@ -144,7 +144,10 @@ export async function executePostBundleSteps(
         workspaceRoot,
         serviceWorker,
         options.baseHref || '/',
-        outputFiles,
+        // Ensure additional files recently added are used and also only browser files
+        [...outputFiles, ...additionalOutputFiles].filter(
```

Comment:
Actually this will cause media files not to be included.

https://github.com/angular/angular-cli/blob/b9b0fb9580f39d4731eb845da279a0fd16f40804/packages/angular_devkit/build_angular/src/utils/service-worker.ts#L69-L86

You can potentially just remove the `filter` from here since it's already handled at a latter stage.